mod/layout/set/abstract/thumbnail.rb

```diff
@@ -1,6 +1,7 @@
 format :html do
-  view :thumbnail_plain do
-    wrap_with :div, thumbnail_content, class: flex_css
```

Comment:
I couldn't figure out enough of a difference between `thumbnail` and `thumbnail plain` to document it, so I merged them.  I did see that thumbnail didn't have all the "data" in the div, but that hardly seemed worth a separate view.